lib/common/http1client.c

```diff
@@ -595,6 +610,17 @@ static void on_whole_request_sent(h2o_socket_t *sock, const char *err)
     }
 }
 
+static void on_header_sent(h2o_socket_t *sock, const char *err)
```

Comment:
Maybe rename the function to `on_headers_sent_wait_100` or something?

Current name might be confusing in sense that it sounds as if this function is always called after the request headers have been sent.

Reply:
https://github.com/h2o/h2o/pull/3316/commits/3c7b85016a17ccee04ef5d0b76f80e92b02225e2